src/legacy/core_plugins/kibana/public/discover/angular/context/query_parameters/__tests__/action_add_filter.js

```diff
@@ -19,15 +19,16 @@
 
 import expect from '@kbn/expect';
 import ngMock from 'ng_mock';
-import { getServices } from '../../../../kibana_services';
+import { FilterBarQueryFilterProvider } from '../../../../kibana_services';
```

Comment:
Please use `data.query.filterManager` - `FilterBarQueryFilterProvider` is a wrapper around it.

Reply:
done, thx!